grouper/email_util.py

```diff
@@ -256,3 +256,56 @@ def notify_edge_expiration(settings, session, edge):
         settings=settings,
         context=email_context,
     )
+
+
+def notify_nonauditor_flagged(settings, session, edge):
+    """Send notification that a nonauditor in an audited group has had their membership
+    set to expire.
+
+    Handles email notification and audit logging.
+
+    Args:
+        settings (Settings): Grouper Settings object for current run.
+        session (Session): Object for db session.
+        edge (GroupEdge): The expiring edge.
+    """
+    # TODO(herb): get around circular depdendencies; long term remove call to
+    # send_async_email() from grouper.models
+    from grouper.model_soup import Group
+    from grouper.models.base.constants import OBJ_TYPES_IDX
+    from grouper.models.user import User
+
+    # TODO(rra): Arbitrarily use the first listed owner of the group from which membership expired
+    # as the actor, since we have to provide an actor and we didn't record who set the expiration on
+    # the edge originally.
+    actor_id = next(edge.group.my_owners().itervalues()).id
```

Comment:
I was originally leaning towards having it be the same user, but chose to do this since rra knows best. I think of the 3 options, having it be the same user is the best choice.